apps/mark/frontend/src/pages/poll_worker_screen.tsx

```diff
@@ -495,18 +509,18 @@ export function PollWorkerScreen({
   const [isDiagnosticsScreenOpen, setIsDiagnosticsScreenOpen] = useState(false);
   const [scannerReportData, setScannerReportData] =
     useState<ScannerReportData>();
+
   useEffect(() => {
-    if (scannerReportData) return;
-    if (!pollworkerAuth.card.hasStoredData) return;
```

Comment:
Got rid of the `hasStoredData` field in new models since, in Java Card land, the best way to populate such a boolean would just be to try to read data. And as you'll see below anyway, I got rid of the need to even check `hasStoredData` here

Reply:
That makes sense